kafka-connect-upload-source/Dockerfile

```diff
@@ -30,10 +30,10 @@ COPY --from=builder /code/kafka-connect-upload-source/build/third-party/*.jar ${
 COPY --from=builder /code/kafka-connect-upload-source/build/libs/kafka-connect-upload-source-*.jar ${CONNECT_PLUGIN_PATH}/kafka-connect-upload-source/
 
 # Load topics validator
-COPY ./docker/kafka-wait /usr/bin/kafka-wait
+COPY ./kafka-connect-upload-source/src/main/docker/kafka-wait /usr/bin/kafka-wait
```

Comment:
I think the `kafka-connect-upload-source` should be removed here, if the Dockerfile sits inside that folder?

Reply:
Because the build context is the root folder, copy operations still need to include this unfortunately. This is one of the limitations of the Docker build system and context.